.github/workflows/doc-build.yml

```diff
@@ -45,8 +45,9 @@ jobs:
     - name: install-pip
       run: |
         pip3 install setuptools wheel
-        pip3 install -r scripts/requirements-base.txt
         pip3 install -r scripts/requirements-doc.txt
+        # Building without west is also possible, see exclude_patterns
+        # in conf.py
```

Comment:
I don't think we want to build without west in CI, so this comment may be misleading here

Reply:
The "official" stance is that building the docs without west is not tested in CI but it is officially supported (and works last time I checked). So it is worth mentioning that one of the `requirements` is not actually a strict requirement.

Reply:
This information belongs to documentation build guidelines, but definitely not on CI scripts, where `west` is always desired as a requirement.

Reply:
Done!

PS: when I want to build something not obvious I always look at CI scripts before looking at documentation; the latter is always lagging behind whereas the CI scripts are by nature always working. Plus this shortcut was not "officially supported", it'll never make it to the documentation.